### ISSUE TITLE
[Date picker] Calendar should start the week with Sundays in Arabic

### ISSUE THREAD

**Issue by yannikmesserli** [Esri/calcite-components#7041]
### Actual Behavior

In `ar` locale, the input date picker starts the week on Saturdays

<img width="1123" alt="Screenshot 2023-05-26 at 13 54 13" src="https://github.com/Esri/calcite-components/assets/1910449/196c2092-12d7-41d9-b358-93369ffe13c2">

### Expected Behavior

It should start on Sundays

### Reproduction Sample

https://jsbin.com/qiqikor

### Reproduction Steps

- Open repro app
- Click on input
- See the calendar

### Reproduction Version

Tested in 1.0.8

### Priority impact

p3 - want for upcoming milestone

### Esri team

ArcGIS Urban

**Comment by jcfranco** [comment]
I believe this is working as expected:

* https://github.com/Esri/calcite-components/issues/6182
* https://github.com/Esri/calcite-components/commit/f9fe230ba07d4c581993efacff04303700c07106
* https://github.com/moment/moment/blob/develop/locale/ar.js#L193

cc @annierm18

**Comment by annierm18** [comment]
@jcfranco Yes, that is correct, first day of the week should be Saturday for "ar"

**Comment by geospatialem** [comment]
@yannikmesserli Per the above conversation from Franco and Aine regarding the day of the week starting on Saturday for Arabic, will be closing out this issue.
